src/app/vm/vm-creation/vm-creation.component.html

```diff
@@ -133,6 +129,30 @@ <h4>{{ 'VM_PAGE.VM_CREATION.ZONE' | translate }}</h4>
 
             <mat-tab [label]="'VM_PAGE.VM_CREATION.ADVANCED' | translate">
               <div class="settings-column">
+                <div>
+                  <h4>{{ 'VM_PAGE.VM_CREATION.HOST_NAME' | translate }}</h4>
+                  <mat-form-field floatLabel="never" class="full-width-input">
+                    <input
+                      matInput
+                      name="hostName"
+                      [ngModel]="vmCreationState.name"
+                      (ngModelChange)="hostNameChange.emit($event)"
+                      [placeholder]="'VM_PAGE.VM_CREATION.HOST_NAME' | translate"
+                      pattern="^[a-zA-Z]{1}[a-zA-Z0-9\-]{0,61}[a-zA-Z0-9]$"
+                      [maxlength]="maxEntityNameLength"
+                    >
+                    <mat-hint align="end">
+                      <cs-character-count
+                        [value]="vmCreationState.name"
+                        [maxLength]="maxEntityNameLength"
```

Comment:
`maxEntityNameLength` is too generic. Why do you use `maxEntityNameLength` here and `32` in display name?